Makefile

```diff
@@ -0,0 +1,27 @@
+rel: relc testpy
+
+debug: testc testpy
+
+.PHONY: testc testpy debug relc rel
+
+Debug:
+	mkdir Debug
+
+Debug/Makefile: Debug
+	(cd Debug && cmake .. -DCMAKE_BUILD_TYPE=Debug)
+
+testc: Debug/Makefile
+	(cd Debug && make && make install)
+
+Release:
+	mkdir Release
+
+Release/Makefile: Release
+	(cd Release && cmake .. -DCMAKE_BUILD_TYPE=Release)
+
+relc: Release/Makefile
+	(cd Release && make && make install)
+
+testpy:
+	python setup.py build_ext --inplace
+
```

Comment:
In a future version, would it be possible to have a way of specifyint `-DCMAKE_INSTALL_PREFIX=~/.local` for instance?